index.js

```diff
@@ -1 +1,220 @@
+      litters.add(z);
+      uniqueAnagrams.push(arr[i]);
+    }
+
+  }
+  return uniqueAnagrams;
+
+};
+
+console.log(unique(anagrams));
+
+
+//Getter and setter
+// Create an user object
+// User should have to first name last name and age Add userInfo getter which gets all info from obj
+// Add userInfo setter to set all user info from string or obj
+// String setter format 'Taras Samoilenko 25'
+
+const user = {
+  firstName: "",
+  lastName: "",
+  age: "",
+  get userInfo(){
+    return `${this.firstName} ${this.lastName} is ${this.age}`
+  },
+  set userInfo(a){
+    if (typeof(a) === 'string') {
+      let arrWords = a.split(" ");
+      this.firstName = arrWords[0];
+      this.lastName = arrWords[1];
+      this.age = +arrWords[2];
+    } else if (typeof(a) === 'object') {
+      this.firstName = a.firstName;
+      this.lastName = a.lastName;
+      this.age = a.age;
+    }
+  },
+};
+
+user.userInfo = "Taras Samoilenko 25";
+user.userInfo;
+// "Taras Samoilenko is 25"
+
+user.userInfo = { firstName: "Kate", lastName: "Karp", age: 22 };
+user.userInfo;
+// "Kate Karp is 22"
+
+
+//Create arr method
+// Create obj with property from and to
+// Create function createArr that reads obj properties and
+// Creates new sorted arr with range from and to
+// Assign this arr as obj property
+
+// Create 3 solutions Solution 1 without bind, call or apply
+// Solution 2 with bind Solution 3 with call or apply
+
+const obj = {
+  from: 1,
+  to: 10,
+};
+
+const createArr = function (){
+  let arrFromTo = [];
+  for (let i = this.from; i <= this.to; i++){
+    arrFromTo.push(i);
+  };
+  return arrFromTo;
+};
+
+// Sol 1
+obj.createArr = createArr;
+
+// Sol 2
+const bindedSolution = createArr.bind(obj);
+
+// Sol 3
+const callSolut = createArr.call(obj)
+
+
+//Constructor Function
+// Create a constructor function that creates car obj
+// Obj should have properties model, color, age, speed, gasTank and started 
+// Obj should have methods startEngine, drive, stop, speedUp, slowDown, addGas This methods should be chainable
+// startEngine method checks if car has gas (you can create another method for checking gas)
+// If gas is not empty set property started to true
+// drive method should do smth only if property started is true and gas is not 0
+// if started true increase speed to 30
+// stop method sets property started to false and speed to 0
+// speedUp method increases speed by arg. Max speed is 200
+// Each speedUp method decreases gas by 5. Max gas is 20 If gas is empty stop car
+// gas cannot be less then 0
+// If started is false stop car
+// speedUp(5) adds 5 to speed
+// slowDown works like speedUp but decreases speed. Min speed is 0
+// addGas method adds gas to car by arg
+// addGas(5) adds 5 to gas
+
+function Car(model, color, age, speed, gasTank, started) {
+  this.model = model;
+  this.color = color;
+  this.age = age;
+  this.speed = speed;
+  this.gasTank = gasTank;
+  this.started = started;
+  function startEngine (){
```

Comment:
move this function out of Car constructor or assign it directly to method

` this.startEngine = function () {
   // code
  };`